CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed a bug where sometimes tensorboard logger may be not closed properly. Making `Trainer` a context manager.
```

Comment:
Just a nit-pick here but this should really be two different CHANGELOG entries.

Something like this should be in the `### Added` section: "Added a `Trainer.close()` method to close / clean up any open resources the trainer is using, which will be automatically called when the `Trainer` is used as a context manager"

And then the bug fix note in the `### Fixed` section like you have it.

Reply:
Sorry about that.
Just added to the changelog.